doc/content/verification_validation_examples/slab_benchmark/model/solid.i

```diff
@@ -0,0 +1,217 @@
+T0 = 293 # K
+L0 = 100 # cm
+L = 106.47 # cm
+P = 1.0e22 # eV/s
+q = 1e8 # eV
+k0= 1.25e19 # eV/(s-cm-K^2) k(T) = k0 T(x)
+phi0 = 2.5e14 # 1/s-cm^2 (flux at the origin)
+eV_to_J = 1.602e-19 # J/eV
+lam = ${fparse 0.5*(1+sqrt(1+(16*q*q*phi0*phi0)/(P*P)))} # eigenvalue solution
+h = ${fparse 1/(sqrt(L*(lam-1)/(k0*P)) - (2*T0)/(P))*eV_to_J } # W/cm^2-K
+
+[Mesh]
+  [generated]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = 20
+    xmin = -50
+    xmax = 50
+  []
+  [pin]
+    type = ExtraNodesetGenerator
+    input = generated
+    new_boundary = pin
+    coord = '0 0 0'
+  []
+[]
+
+[GlobalParams]
+  displacements = 'disp_x'
+  use_displaced_mesh = true
+[]
+
+[Variables]
+  [temp]
+  []
+  [disp_x]
+  []
+[]
+
+[Physics]
+  [SolidMechanics]
+    [QuasiStatic]
+      [block1]
+        strain = FINITE
+        add_variables = true
+        eigenstrain_names = eigenstrain
+        generate_output = 'strain_xx'
+        temperature = temp
+      []
+    []
+  []
+[]
+
+[ICs]
+  [temp]
+    type = FunctionIC
+    variable = temp
+    function = 293
+  []
+[]
+
+[Kernels]
+  [heat_conduction]
+    type = HeatConduction
+    variable = temp
+  []
+  [kappa_fission]
+    type = CoupledForce
+    variable = temp
+    v = kappa_fission
+  []
+  [dt]
+    type = HeatConductionTimeDerivative
+    variable = temp
+  []
+[]
+
+[AuxVariables]
+  [kappa_fission]
+    family = MONOMIAL
+    order = CONSTANT
+    initial_condition = '${fparse P*eV_to_J/L0 }'
+  []
+  [temp_praux]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [disp_praux]
+  []
+[]
+
+[AuxKernels]
+   [temp_projection]
+     type = ProjectionAux
+     variable = temp_praux
+     v = temp
+   []
+   [disp_projection]
+     type = ProjectionAux
+     variable = disp_praux
+     v = disp_x
+   []
+[]
+
+[Materials]
+  [thermal_parameters]
+    type = HeatConductionMaterial
+    temp = temp
+    thermal_conductivity_temperature_function = conductivity
+    specific_heat = 0.3
+  []
+  [density]
+    type = Density
+    density = 20.0
+    use_displaced_mesh = false
+  []
+  [elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 1e6
+    poissons_ratio = 0.0
```

Comment:
I suggest adding these numbers at the top of the file and provide units for them.